backend/src/test/java/com/redhat/cloud/notifications/routers/sources/SecretUtilsTest.java

```diff
@@ -88,11 +96,15 @@ void getSecretsForEndpointTest() {
         final String secretToken = props.getSecretToken();
         Assertions.assertEquals(SECRET_TOKEN, secretToken, "the secret token doesn't match");
 
+        final String secretBearer = props.getBearerAuthentication();
+        Assertions.assertEquals(BEARER_TOKEN, secretBearer, "the secret bearer doesn't match");
+
         // Assert that the underlying function was called exactly two times, since we are expecting that both the
         // "basic authentication" and the "secret token" secrets were fetched.
```

Comment:
```diff
-        // Assert that the underlying function was called exactly two times, since we are expecting that both the
-        // "basic authentication" and the "secret token" secrets were fetched.
+        // Assert that the underlying function was called exactly three times,
+        // since we are expecting that the "basic authentication", "secret
+        // token" and "bearer authentication" secrets were fetched.
```